storm-core/src/clj/backtype/storm/daemon/executor.clj

```diff
@@ -663,8 +673,9 @@
                     (reset! last-active true)
                     (log-message "Activating spout " component-id ":" (keys task-datas))
                     (fast-list-iter [^ISpout spout spouts] (.activate spout)))
-               
-                  (fast-list-iter [^ISpout spout spouts] (.nextTuple spout)))
+
+                  (fast-list-iter [^ISpout spout spouts] (.nextTuple spout))
```

Comment:
I'm not sure we want to split the ')' onto a new line.  It does not follow the formatting.